presto-hive/src/main/java/io/prestosql/plugin/hive/HiveApplyProjectionUtil.java

```diff
@@ -102,6 +102,10 @@ public static ConnectorExpression replaceWithNewVariables(ConnectorExpression ex
 
     /**
      * Returns the assignment key corresponding to the column represented by {@param projectedColumn} in the {@param assignments}, if one exists.
+     * The variable in the {@param projectedColumn} can itself be a representation of another projected column. For example,
```

Comment:
Does this comment reflect new behavior? If not, move it to the commit that introduced the method.